src/components/connectAdvanced.js

```diff
@@ -374,9 +374,7 @@ export default function connectAdvanced(
         const unsubscribeWrapper = () => {
           didUnsubscribe = true
           subscription.tryUnsubscribe()
-          if (window.react_redux_shouldClearOnStateChange) {
-            subscription.onStateChange = null
-          }
+          subscription.onStateChange = null
```

Comment:
The've merged my pull request to their master so no need for the react_redux_shouldClearOnStateChange

Reply:
Cool